File: src/main/java/myLang/expression/ExpressionContext.java
package myLang.expression;

import myLang.ContextHandler;
import myLang.response.*;
import myLangParser.MyLangParser;

public class ExpressionContext implements ContextHandler<MyLangParser.ExpressionContext> {
    @Override
    public Response handler(MyLangParser.ExpressionContext ctx) throws MyLangException {
        return defaultHandler(ctx);
    }
}

File: src/main/kotlin/myLangKotlin/expression/ExpressionContext.kt
package myLangKotlin.expression

import myLangKotlin.ContextHandler
import myLangKotlin.response.MyLangException
import myLangKotlin.response.Response
import myLangParser.MyLangParser

class ExpressionContext : ContextHandler<MyLangParser.ExpressionContext> {
    @Throws(MyLangException::class)
    override fun handler(ctx: MyLangParser.ExpressionContext): Response<*> {
        return defaultHandler(ctx)
    }
}

